src/Routes/index.js

```diff
@@ -32,21 +33,34 @@ const Routes = () => (
     <PrivateRoute path="/profile" component={ProfileView} />
     <PrivateRoute path="/study/:kfId(SD_\w{8})/" component={NavBarView} />
     <AdminRoute path="/study/new-study" component={NewStudyView} />
-    <Switch>
-      <PrivateRoute exact path="/" component={StudyListView} />
-      <PrivateRoute path="/study/:kfId/basic-info" component={StudyInfoView} />
-      <PrivateRoute exact path="/study/:kfId/dashboard" component={EmptyView} />
-      <PrivateRoute path="/study/:kfId/cavatica" component={CavaticaBixView} />
-      <PrivateRoute exact path="/study/:kfId/logs" component={LogsView} />
-      <AdminRoute exact path="/tokens" component={TokensListView} />
-      <AdminRoute
-        exact
-        path="/cavatica-projects"
-        component={CavaticaProjectsView}
-      />
-      <AdminRoute exact path="/events" component={EventsView} />
-      <DocumentRoutes />
-    </Switch>
+    <div className="page">
```

Comment:
looks like `ProfileView` and `NewStudyView` ☝️ would be left out of the wrapper?